README.md

```diff
@@ -26,15 +28,18 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+We use data-ng-click for validation purposes especially if we are working with API.
```

Comment:
:+1: for the validation part, but not sure why it helps when working with APIs...